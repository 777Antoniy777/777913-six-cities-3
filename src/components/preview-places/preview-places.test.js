import React from "react";
import renderer from "react-test-renderer";
import PreviewPlaces from "./preview-places";

// set mocha data
const offer = {
  id: 1,
  city: {
    id: 1,
    name: `city`,
    coords: [1, 1],
  },
  title: `title 1`,
  premium: false,
  src: `img/image1`,
  photos: [`img/image1`],
  price: 999999,
  description: `test`,
  type: `type`,
  rating: 9999,
  bedroomAmount: 30,
  guestsAmount: 50,
  items: [`item`],
  host: {
    avatar: `img/avatar-1.jpg`,
    name: `name`,
    status: false,
  },
  reviews: [
    {
      id: 1,
      body: `text`,
      rating: 5,
      name: `name`,
      date: `date`,
    },
  ],
  coord: [1, 1],
};
const history = {};
const location = {
  pathname: `/pathname`,
};

const getActiveItem = () => {};

it(`render PreviewPlaces`, () => {
  const tree = renderer.create(
      <PreviewPlaces
        offer={offer}
        history={history}
        location={location}
        getActiveItem={getActiveItem}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
