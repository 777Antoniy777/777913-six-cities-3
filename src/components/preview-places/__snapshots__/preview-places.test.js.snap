// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PreviewPlaces 1`] = `null`;
